dox/books/Reference_Manual/main_pdf.dox

```diff
@@ -1,18 +1,12 @@
 /**
-@mainpage   Bill
+@mainpage   Bill Reference Manual
 
-@htmlonly
-<h1>Reference Manual</h1>
+<h1>Bill Reference Manual</h1>
 <h4>Introduction</h4>
-:
-    - :
-    <ul>
-        <li> 
-    </ul>
+TBD...
 
 <h4>License</h4>
```

Comment:
Please remove License notification entirely.

For explanation, please see comment on the Primer's main_pdf.dox.